src/rencache.c

```diff
@@ -200,7 +201,7 @@ static void push_rect(RenRect r, int *count) {
 }
 
 
-void rencache_end_frame(lua_State *L) {
+void rencache_end_frame(UNUSED lua_State *L) {
```

Comment:
Ditto.

Reply:
removing it moved the UNUSED from this to `f_end_frame`